.github/workflows/create-release.yml

```
@@ -0,0 +1,85 @@
+jobs:
+  create-release:
+    name: Create tag and GitHub Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v6
+        with:
+          fetch-depth: 0
+
+      - name: Get version
+        id: version
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:6:28: Double quote to prevent globbing and word splitting

(shellcheck)